### DISCUSSION TITLE
帮忙生成一个应用

### DISCUSSION THREAD

**Discussion by weqoocu** [tw93/Pake#833] [Q&A]
https://marker.dotalk.cn/#/daily 

换了新电脑，无法生成。 求助

**Comment by weqoocu** [comment]
fork后，github也没有生成
<img width="882" alt="image" src="https://github.com/user-attachments/assets/e6421ef3-31ed-4983-8c7c-4ef67a75abd8">

**Comment by Tlntin** [comment]
你的forked的代码看着有点旧了，可以重新fork一下，还有应用名不能为中文，建议小写字母。
